From: Shengjing Zhu <zhsj@debian.org>
Date: Sun, 23 Aug 2020 18:58:06 +0800
Subject: skip TestSystemd240

Forwarded: not-needed
---
 cgroup1/paths_test.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cgroup1/paths_test.go b/cgroup1/paths_test.go
index 8afe2ce..8646821 100644
--- a/cgroup1/paths_test.go
+++ b/cgroup1/paths_test.go
@@ -92,6 +92,7 @@ func TestRootPath(t *testing.T) {
 }
 
 func TestSystemd240(t *testing.T) {
+	t.Skip("cgroup is not mounted in Debian schroot")
 	if isUnified {
 		t.Skipf("requires the system to be running in legacy mode")
 	}
